### PR TITLE
#167709673:Updating feedback on html Home Page

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
 - Updating feedback on html Home Page
#### Description of Task to be completed?
*Home page*
- The files inside the folder UI folder should be in lower case
- The Menu bar should be arranged with a search on it
- The footer should be Fixed
- Na empty space or Unwanted comments
- The photo Should be arrange 2-2 on tablet then 4 on the desktop
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167709673

#### Screenshots (if appropriate)

home page change
home page change2
home page change3

### CODE REVIEW

File: UI/login.html [hunk 1]
```diff
@@ -22,7 +22,6 @@
       </ul>
     </nav>
   </header>
-
   <div class="login-page">
```
Comment (oesukam) [inline UI/login.html]: This space is not necessary

**Review by oesukam — CHANGES_REQUESTED** [review]

**Comment by oesukam** [comment]
@Niyonsengaeric all your pages names should be in lowercase

**Comment by oesukam** [comment]
@Niyonsengaeric Please take this time to restructure the project before doing anything else

**Comment by oesukam** [comment]
@Niyonsengaeric Please follow the branch naming convention. `ch-homePageFeedback-167709673` should instead be `ch-home-page-feedback-167709673`

**Comment by Niyonsengaeric** [comment]
OK let me correct it

File: UI/login.html (hunk 1 above)
Comment (Niyonsengaeric) [reply UI/login.html]: i have remove it

**Review by oesukam — APPROVED** [review]

**Pull request merged**